TESTS/usb_device/basic/USBTester.h

```diff
@@ -17,6 +17,7 @@
 
 #ifndef USB_TESTER_H
 #define USB_TESTER_H
+#if defined(MBED_CONF_RTOS_PRESENT)
```

Comment:
Are you sure it is necessary to compile this out? I think it might or should not be included by the source files of there is no RTOS.